structs.go

```diff
@@ -267,3 +267,27 @@ func marshalFqCodelXStats(v *FqCodelXStats) ([]byte, error) {
 	_, err2 := buf.Write(subStat)
 	return buf.Bytes(), concatError(err, err2)
 }
+
+// FqQdStats from include/uapi/linux/pkt_sched.h
+type FqQdStats struct {
+	GcFlows             uint64
+	HighPrioPackets     uint64
+	TcpRetrans          uint64
```

Comment:
Can we update this name?
```suggestion
	TCPRetrans          uint64
```